web-ui/src/main/resources/catalog/components/userfeedback/partials/userfeedbacklasthome.html

```diff
@@ -0,0 +1,32 @@
+<div>
+
+  <div data-ng-show="lastCommentsList.length">
+    <div class="gn-userfeedback-card"
+      data-ng-repeat="row in lastCommentsList">
+      <div class="gn-userfeedback-pre-title" data-translate="">GUFfeedbackOn</div>
+      <h4>
+        <a data-ng-href="#/metadata/{{row.metadataUUID}}">{{row.metadataTitle}}</a>
+      </h4>
+      <q class="gn-quote" data-ng-show="row.comment">{{row.comment}}</q>
+      <div class="clearfix">
+        <div
+          class="gn-userfeedback-score gn-userfeedback-score-{{row.ratingAVG}} gn-userfeedback-badge"
+          data-ng-show="row.ratingAVG">
+          {{row.ratingAVG}} / 5 <span class="fa fa-fw fa-star">&nbsp;</span>
+        </div>
+        <div class="gn-userfeedback-score" data-ng-hide="row.ratingAVG">
+          - / - <span class="fa fa-fw fa-star">&nbsp;</span>
+        </div>
+        <div class="gn-userfeedback-reply">
+          <a>{{row.authorName}}</a><br>on {{row.date | date:'medium'}}
```

Comment:
Link on user is not set - should we remove ?

At some point, we may group new messages on a same record ?
![image](https://user-images.githubusercontent.com/1701393/38680214-04fcd8d8-3e66-11e8-87a8-cf1e7861911a.png)